src/routes/MemberProfile/MemberProfile.js

```diff
@@ -218,6 +225,33 @@ export function ActionDropdown ({ items }) {
     />
 }
 
+export function ProjectsSection ({ projects, showDetails }) {
+  return (
+    <div>
+      <div styleName='profile-subhead'>
+        Projects
+      </div>
+      {projects.map((p, index) => {
+        return (<Project key={index} project={p} showDetails={showDetails} />)
+      })}
+    </div>
+  )
+}
+
+export function Project ({ project, showDetails }) {
+  const { title, id, createdAt, creator, members } = project
+  const MEMBER_CAP = 3
```

Comment:
Trivial note, but I would make this a prop with a default value, just because I think that'd be a little more clear, testable, and suddenly configurable.